packages/runtime/container-runtime/src/dataStore.ts

```diff
@@ -159,6 +161,7 @@ class DataStore implements IDataStore {
         private readonly logger: ITelemetryLogger,
     ) {
         this.pendingAliases = datastores.pendingAliases;
+        this.handle = fluidDataStoreChannel.handle;
```

Comment:
My understanding is that the `handle` wouldn't change once it is initialized, so this doesn't need to be a getter that returns the `fluidDataStoreChannel.handle` every time.